### DISCUSSION TITLE
Add a web login module

### DISCUSSION THREAD

**Discussion by smallcoderhujin** [arabcoders/ytptube#110] [General] [answered]
It is recommended to add a login module that supports setting the username and password through environment variables.

**Comment by arabcoders** [comment, marked as answer]
Hi,

It's something i would like to explore as things stand right now it's possible to do it via a proxy manager for example, 
https://github.com/linuxserver/reverse-proxy-confs/blob/63958167f1fa4befc60a5e33edb81c39932ac5db/watchstate.subdomain.conf.sample


Edit:

here you go https://github.com/arabcoders/ytptube?tab=readme-ov-file#authentication
